userspace/libsinsp/event.h

```diff
@@ -31,11 +31,11 @@ typedef class sinsp_threadinfo sinsp_threadinfo;
 ///////////////////////////////////////////////////////////////////////////////
 typedef enum filtercheck_field_flags
 {
-	EPF_NONE = 0,
```

Comment:
I don't think there's anything that prevents the creation in the future of a field such as `EPF_TABLE_ONLY | EPF_REQUIRES_ARGUMENT`, in what sense they seem exclusives?

Anyway, the goal of the patch is to fix a real bug: all those enum values are then checked all over the place with `&`. For example, `evt.rawarg` is marked as `EPF_REQUIRES_ARGUMENT` (which evaluates to 3), so when the code iterates over the filters and checks for `& EPF_FILTER_ONLY` (which evaluates to 1), that field will wrongly be reported as "filter only", which is non sense since that field can be printed as it's not ambiguous.

Moving to proper bit flags seems to fix this behavior with minimal changes in the code and allows multiple concatenation of filter flags for the same field (first point), which seems like a good thing as those do not necessarily look semantically exclusive to me.